api/docgen/openrpc.go

```diff
@@ -56,7 +56,12 @@ func ParseCommentsFromNodebuilderModules(moduleNames ...string) Comments {
 	fset := token.NewFileSet()
 	nodeComments := make(Comments)
 	for _, moduleName := range moduleNames {
-		f, err := parser.ParseFile(fset, "nodebuilder/"+moduleName+"/service.go", nil, parser.AllErrors|parser.ParseComments)
+		f, err := parser.ParseFile(
+			fset,
+			"nodebuilder/"+moduleName+"/"+moduleName+".go",
```

Comment:
[JUST A NIT]: I am generally in favor of doing one thing per line, and in this case, constructing the string using `fmt.Sprintf` somewhere prior to using it in `ParseFile`.

Reply:
done